packages/communication-react/CHANGELOG.stable.md

```diff
@@ -7,21 +7,27 @@
 ## [1.27.0](https://github.com/azure/communication-ui-library/tree/1.27.0)
 
 Mon, 12 May 2025 20:47:06 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/1.27.0-beta.1...1.27.0)
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.26.0...1.27.0)
 
 ### Features
-- move onfetchProfile to stable ([PR #5796](https://github.com/azure/communication-ui-library/pull/5796) by 97124699+prabhjot-msft@users.noreply.github.com)
-- Add onFetchProfile for CallWithChat Composite ([PR #5729](https://github.com/azure/communication-ui-library/pull/5729) by 97124699+prabhjot-msft@users.noreply.github.com)
-- Fix listeners subscribed to chat events of AzureCallWithChatAdapter when the thread id is resolved late or is changed when moving to and from breakout rooms ([PR #5764](https://github.com/azure/communication-ui-library/pull/5764) by 79475487+mgamis-msft@users.noreply.github.com)
-- Ensure text wrap for long words without spaces ([PR #5789](https://github.com/azure/communication-ui-library/pull/5789) by 96077406+carocao-msft@users.noreply.github.com)
+
+#### onFetchProfile
```

Comment:
We also need an item here for Together mode

Reply:
Did not see any changes related to together mode, are there any PRs?